### ISSUE TITLE
a plugin that's not a plugin?

### ISSUE THREAD

**Issue by dan-da** [AdamISZ/electrum-joinmarket-plugin#34]
I was following along the install instructions until I came to:

> Download the customised Electrum release from this repo

say what?   I thought this was supposed to be a plugin.  Why install a custom electrum exactly?   not really something I'm willing to do....

**Comment by AdamISZ** [comment]
As of today, i wouldnt install it at all; i believe there's a note to that effect in the readme?

Wrt custom version,  there were two things: from source, you can add to plugins directory but it needs a number of extra steps (mainly because of extra dependencies). When it came to binaries  there was no avoiding creating custom version since thomasv is 100% clear he will include any coinjoin plugin, this or other. Everyone is annoyed that i dont create a 1 click install and refuses to consider using it otherwise. Similarly everyone gets upset if i create something custom packaged as it doesnt inherit electrum's security. And, no one helps me test it. So frankly I cant really be bothered any more, until that changes. (sorry havent checked exact details right now, on plane).

**Comment by AdamISZ** [comment]
Ok just while i have conn, you can see diff in "from source" version.

**Comment by dan-da** [comment]
sorry I did not research about electrum plugins first.   When I hear the word plugin I generally assume that a third party can supply software modules externally from the program author.   From your comment I guess that is not the case with Electrum...?  or I'm not exactly sure what you mean by a 1 click install.

If electrum can only use plugins that are vetted and approved by program author then that is disappointing as it kind of defeats purpose of a plugin system, imho.   also sad that he won't support enhanced privacy for electrum users via coinjoin/jm...

**Comment by AdamISZ** [comment]
Yes, you have it right, sorry for the slightly .. tetchy tone, I guess you hit a bit of a raw nerve :)

But as I said in the brief second comment, the setup that you were envisaging is available, in the second "From source" (not "Quick version") in the readme; i.e. install electrum then install these plugin files; but it has to be from source, following those extra steps, it can't be in a binary distribution until/unless some more details are worked out. And yes, Electrum's plugin architecture is more conceived as offering support to developers who then get it merged into the Electrum project itself, not so much what I was trying to do (because of the mentioned restriction, was trying to do it outside Electrum's own project).

And finally to reiterate development (at least by me!) has, for now, stopped on this, it has not been updated for segwit for example; as per the note in the readme, it may still be possible to use older versions, but no guarantees. It needs attention and for now it isn't getting it.
